packages/code-snippet/src/CodeSnippetService.ts

```diff
@@ -47,7 +51,7 @@ export class CodeSnippetService {
   }
 
   // TODO: Test this function
-  async findByLanguage(language: string): Promise<ICodeSnippet[]> {
+  static async findByLanguage(language: string): Promise<ICodeSnippet[]> {
```

Comment:
this is not being used anywhere

Reply:
Hm yeah I think that was already not being used - do you remember why that was added in the first place? Maybe for future searching capabilities in the code snippet display?

Reply:
Yeah I think that was the intent, we can keep it there then :)